app/assets/javascripts/modules/moj.AsyncGA.js

```diff
@@ -13,9 +13,9 @@
       window.ga('create', gaTrackingId, 'service.gov.uk');
 
       if (hitTypePage) {
-        window.ga('send', 'page_view', location.pathname + '#' + hitTypePage);
+        window.ga('send', 'pageview', location.pathname + '#' + hitTypePage);
```

Comment:
?? huh that's odd I remember fixing that a little while ago. Could we add test around this before it breaks again?